SDK-315 [Node.js SDK] Add information that sorting options will be working only in combination with the non-hashing Storage mode (#188)
incountry/sdk-nodejs: README.md

- `offset` allows to specify the starting index used for records pagination;
- `sort` allows to sort the returned records by one or multiple keys;

> **WARNING** To use `sort` with find for string keys `key1...key20` you need `hashSearchKeys` option to be set to `false`

Reviewer: ```suggestion
**WARNING** To use `sort` in find() call for string keys `key1...key20` you need to set  `hashSearchKeys` option to `false`.
```
